ADALiOS/ADALiOS/ADAuthenticationParameters.h

```diff
@@ -35,7 +35,7 @@
 /*! The resource URI, as returned by the server. */
 @property (readonly) NSString* resource;
 
-@property (readonly, getter = getExtractedParameters) NSDictionary* extractedParameters;
+@property (readonly, getter = extractedParameters) NSDictionary* extractedParameters;
```

Comment:
You can remove the "getter = _____" part entirely here, that's the default value.